pyproject.toml

```diff
@@ -48,13 +48,15 @@ pre-commit = "^2.13.0"
 pytest-aiohttp = "^1.0.4"
 time-machine = "^2.6.0"
 Jinja2 = "<3.1.0"
+syrupy = "^3.0.0"
 
 [tool.black]
 includes = "src"
 
 [tool.pytest.ini_options]
 asyncio_mode = "auto"
 testpaths = ["tests"]
+addopts = "--strict-markers"
```

Comment:
IMO this should be the default - if you use an undefined marker it'll fail fast - good for catching typos.